requirements.txt

```diff
@@ -2,4 +2,5 @@ requests==2.27.1
 python-dotenv==0.20.0
 pytest==7.1.1
 pytest-mock==3.7.0
-responses==0.20.0
\ No newline at end of file
+responses==0.20.0
```

Comment:
@cksnp Do we care about dependencies that are out of date? is it worth using `>=` admittedly this can make things a bit fragile or we can use a dependency to check and monitor the external dependencies versions. wdyt?

Reply:
Also, do we wish to use coverage checks? We might want to block merge until reviewed.

Reply:
> @cksnp Do we care about dependencies that are out of date? is it worth using `>=` admittedly this can make things a bit fragile or we can use a dependency to check and monitor the external dependencies versions. wdyt?

I'm hesitant about dependencies and using `>=` primarily because of fear of supply chain attacks (what you see often happening in the nodejs world). Not that we are fully shielded now, as we could have transitive dependencies auto-updating, but it's something. The real solution is to add Snyk into the mix and have it suggest updates, like we do in the frontend. I know GH has a similar service, but Snyk would be useful also from the security point of view. I'll do that.

Reply:
> Also, do we wish to use coverage checks? We might want to block merge until reviewed.

Yup, I did mean to add coverage checks after the tests were added, but couldn't find the time. Would you be so kind to take care of this?